### PR TITLE
LRQA-53877 | Master

### PR DESCRIPTION
**Pull request by nkkwon**

### CODE REVIEW

**Comment by alexcandidos** [comment]
ci:test:app-builder

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -1150,6 +1150,45 @@ definition {
...
+		AppBuilderAdmin.gotoBack();
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object 3",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.gotoBack();
+
+		AppBuilderAdmin.searchByItem(itemName = "object");
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hey @nkkwon could you please assert if the itemName is being found or not through validateIfSearchItemIsFound macro?

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 2]
```diff
@@ -1150,6 +1150,45 @@ definition {
...
+
+		AppBuilderAdmin.chooseSearchFilterByOption(filterOption = "Name");
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hey @nkkwon could you please validate if the objects are being changing the order when switching between filter option by Name and Modified Date ? You can use the macro AppBuilderAdmin.validateListTitleAtRow().

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 3]
```diff
@@ -1150,6 +1150,45 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Search for an object in the object list"
+	@priority = "5"
+	test SearchForObject {
+		property portal.acceptance = "true";
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hey @nkkwon , could you please change it to "false"? I don't want to have it as an acceptance/relevant test case. Thanks!

**Comment by nkkwon** [comment]
Pushed up all new changes, please let me know if there is anything else to modify.

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase (hunk 2 above)
Comment (alexcandidos) [reply modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hey @nkkwon , I saw that you've added a validation after filter by Modified Date, but It's missing the validation after change the filter by Name.

Comment (nkkwon) [reply modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Pushed new changes for the validation

**Comment by alexcandidos** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/82294. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**